packages/blocks/src/multiple-choice/index.js

```diff
@@ -2,21 +2,28 @@
  * External dependencies
  */
 import { RichText } from '@wordpress/block-editor';
+import { createContext } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import { QuestionWrapper } from '../components';
 
+const Context = createContext( 'multiple-choice' );
```

Comment:
Sidenote: just as a precaution, maybe it'd be a good idea to namespace these with `crowdsignal/...` 🤔